lib/clientv3/bgpconfig.go

```diff
@@ -51,6 +57,10 @@ func (r bgpConfigurations) Create(ctx context.Context, res *apiv3.BGPConfigurati
 		return nil, err
 	}
 
+	if err := r.ValidatePrefixAdvertisements(res); err != nil {
```

Comment:
The reason I added this in clientv3 instead of validator is because we need access to both `sepc.communities` and `spec.prefixAdvertisement.communities` to make sure all the community names passed are pre-defined. 

The validator I added in lib/validator/v3/validator.go are specific to `sepc.communities` and `spec.prefixAdvertisement.communities`, so they don't have access to other fields. 

I see your point, I'll update validator to cover the entire `sepc` of BGPConfig, that way we can have all the validation in one place.